### DISCUSSION TITLE
Self-nomination for NodeJS contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#75] [General]
I'd like to self-nominate to join the `@paketo-buildpacks/nodejs-contributors` team. I've made a handful of contributions to the NodeJS buildpack ecosystem over the past year or so, and I'd like to continue to do so. If we need to see examples I am more than happy to pull some together.

@paketo-buildpacks/nodejs-maintainers can I get a 👍 / 👎  please?

Thanks,
Rob

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on becoming a Node.js contributor!! 🎉
